src/server/routes/reset.ts

```diff
@@ -46,6 +55,17 @@ router.post(
     const { returnUrl, emailSentSuccess } = res.locals.queryParams;
 
     try {
+      if (req.recaptcha?.error) {
+        logger.error(
+          'Problem verifying recaptcha, error response: ',
+          req.recaptcha.error,
+        );
+        throw new ApiError({
+          message: CaptchaErrors.GENERIC,
+          status: 400,
+        });
+      }
```

Comment:
similar to the previous comment, maybe a method which takes `req.recaptcha`, and throws if there's an error?

Reply:
refactored recaptcha initialisation and this into its own method in `recaptcha.ts`

Reply:
https://github.com/guardian/gateway/blob/a7641a7305dfe1fb89bbe1d9dcbeb3299bb01fb3/src/server/lib/recaptcha.ts